src/webapp/common.py

```diff
@@ -148,3 +152,48 @@ def trim_space(s: str) -> str:
 
 def email_to_id(email: str) -> str:
     return hashlib.sha1(email.strip().lower().encode()).hexdigest()
+
+
+def run_git_cmd(cmd: List, dir=None, ssh_key=None) -> bool:
+    if ssh_key and not os.path.exists(ssh_key):
+        raise FileNotFoundError(ssh_key)
```

Comment:
It doesn't look like we catch this exception anywhere so if the ssh key gets deleted accidentally, the app will stop

Reply:
I've got another branch where I'm going through and changing most exceptions to log messages instead. I'll fix it in that branch (if I fix it here, there will be merge conflicts).